exec/src/klio_exec/commands/run.py

```diff
@@ -552,6 +554,42 @@ def _setup_pipeline(self, pipeline):
                     **output_config.to_io_kwargs()
                 )
 
+    def wait_for_pipeline_running(
+        self,
+        result: PipelineResult,
+        timeout_sec=10 * 60,  # 10 minutes in seconds
+        poll_interval_sec=60,
+    ):
+        status = None
+
+        for _ in range(0, timeout_sec, poll_interval_sec):
+            try:
+                status = result.state
+                if status == PipelineState.RUNNING:
+                    logging.info(
+                        "Pipeline status is %s, done waiting", status,
```

Comment:
Nitpick - there's mixing between f-strings and string interpolation for these logs. Mind sticking to f-strings (or str.format)?